features/manage-position/YourPosition.tsx

```diff
@@ -121,37 +121,40 @@ const YourPosition = () => {
           {`${_collateral} ${_collSymbol}`}
         </Status>
         <Status>
-          <Label>Tokens outstanding: </Label>
+          <Label>Tokens created: </Label>
```

Comment:
Why switch this to "created"? I feel like "outstanding" is more intuitive.

Reply:
mainly was the width of the text.

Reply:
I have reverted it to your original text